app/models/collection_change_set.rb

```diff
@@ -20,22 +24,51 @@ def added_reviewers
     point2.reviewers - point1.reviewers
   end
 
+  def removed_managers
+    point1.managers - point2.managers
+  end
+
   def removed_depositors
     point1.depositors - point2.depositors
   end
 
+  def removed_reviewers
+    point1.reviewers - point2.reviewers
+  end
+
+  def participants_changed?
+    added_managers.present? || added_depositors.present? || added_reviewers.present? ||
+      removed_managers.present? || removed_depositors.present? || removed_reviewers.present?
+  end
+
+  def participant_changes
+    {}.tap do |h|
```

Comment:
ah, that's sorta cruft from earlier in my implementation when the caller was asking for `participant_changes` and doing the formatting instead of calling `particpant_change_description`.  will get rid of the extra indirection...